estela-web/src/components/Stats/StatsTableSection.tsx

```diff
@@ -329,35 +328,35 @@ export class StatsTableSection extends Component<DataListSectionProps, DataListS
-                        if (!loadedDatesStats[index]) {
-                            this.retrieveDateJobsStats(index, record.statsDate.jobsMetadata);
-                            return <Spin />;
-                        }
-                        const dataDateTraceStats: StatsTraceabilityDataType[] = jobsDateStats[index].map(
-                            (jobStat: GetJobsStats, jobIndex: number) => {
-                                const status =
-                                    record.statsDate.jobsMetadata.find((jobMeta) => jobMeta.jid === jobStat.jid)
-                                        ?.jobStatus ?? "UNKNOWN";
-                                return {
-                                    key: `${jobIndex}`,
-                                    statsDate: jobStat,
-                                    jobStatus: status,
-                                };
-                            },
-                        );
-                        return (
-                            <Table
-                                className="w-full"
-                                pagination={false}
-                                columns={this.colsTraceabilityTable}
-                                dataSource={dataDateTraceStats}
-                            />
-                        );
-                    },
-                }}
+                // expandable={{
+                //     expandedRowRender: (record, index) => {
+                //         const { loadedDatesStats, jobsDateStats } = this.state;
+                //         if (!loadedDatesStats[index]) {
+                //             this.retrieveDateJobsStats(index, record.statsDate.jobsMetadata);
+                //             return <Spin />;
+                //         }
+                //         const dataDateTraceStats: StatsTraceabilityDataType[] = jobsDateStats[index].map(
+                //             (jobStat: GetJobsStats, jobIndex: number) => {
+                //                 const status =
+                //                     record.statsDate.jobsMetadata.find((jobMeta) => jobMeta.jid === jobStat.jid)
+                //                         ?.jobStatus ?? "UNKNOWN";
+                //                 return {
+                //                     key: `${jobIndex}`,
+                //                     statsDate: jobStat,
+                //                     jobStatus: status,
+                //                 };
+                //             },
+                //         );
+                //         return (
+                //             <Table
+                //                 className="w-full"
+                //                 pagination={false}
+                //                 columns={this.colsTraceabilityTable}
+                //                 dataSource={dataDateTraceStats}
+                //             />
+                //         );
+                //     },
+                // }}
```

Comment:
Is it fine that this is commented out? If this won't be used, let's just remove it.

Reply:
Yes, you're right. It'll be removed in future PRs.